plutus-chain-index/README_marconi.md

```diff
@@ -0,0 +1,159 @@
+
+This is an initial draft for the Marconi README file. Going forward, we will need to discuss and refine this document and/or the implementation so that they match.
+
+# Marconi
+
+## The Cardano blockchain indexer for dApp developers
+
+A lightweight customizable solution for indexing and querying the Cardano blockchain.
+
+### Built by IOG in Haskell
+
+* dApp developers can index whatever is important to them. 
+* Marconi is an indexing solution for developers who need to store on-chain data in a local database. 
+
+## Introduction
+
+Marconi is a blockchain query solution that you can configure for the types of information that you need to index and according to how you need to structure your database schema. 
+
+Marconi consists of these three components: 
+
+| Component | Description | 
+| --------- | ----------- | 
+| Streamer  | Provides event stream of configured data types | 
+| Index     | Indexes and stores the data in a database | 
+| Query     | Fetches information from the database |
+
+The streamer attaches to a node, providing an event stream of the configured data types. Your applications can use streams directly. Or streams may be passed to the index component which indexes and stores the data in a database. You can use the query component to fetch information from the database.
+
+## Design Principles
+
+### High Performance through Customized Solutions for Each Use Case
+
+The philosophy behind Marconi is to maintain high performance by customizing the solution for each use case. You can filter the streams of information to transmit only the data you need for your dApp, minimizing network traffic and database storage. You can also customize database schemas and queries so that the dApps have the most efficient and scalable application-specific API to interact with the data.
+
+### Handling Blockchain Rollback Events Properly
```

Comment:
This does not actually work right now. Indexers will rollback properly, but they do not emit any notifications or events. Maybe this notifying clients of rollbacks is something that the `streaming` library does that I am not aware of.

Reply:
Good to know. For now I will delete the statement about obtaining notifications of advances and rollbacks.